docs/transaction-api.asciidoc

```diff
@@ -49,6 +49,36 @@ The transaction result.
 
 Think of the transaction result as equivalent to the status code of an HTTP response.
 
+[[transaction-ensure-parent-id]]
+==== `transaction.ensureParentId()`
+
+* *Type:* String
+
+If the transaction does not already have a parent id,
+calling this methid generates a new parent id,
+sets it as the parent id of this transaction,
+and returns it as a `String`.
+
+This enables the correlation of the spans the JavaScript Real User Monitoring (RUM) agent creates for the initial page load
+with the transaction of the backend service.
+If your backend service generates the HTML page dynamically,
+initializing the JavaScript RUM agent with the value of this method allows analyzing the time spent in the browser vs in the backend services.
+
+To enable the JavaScript RUM agent,
+add a snippet similar to this to the body of your HTML page,
+preferably before other JS libraries:
+
+[source,js]
+----
+var apm = initApm({
+  pageLoadTraceId: "${transaction.traceId}",
+  pageLoadSpanId: "${transaction.ensureParentId()}",
+  pageLoadSampled: ${transaction.sampled}
+})
+----
+
+See the {apm-rum-ref}[JavaScript RUM agent documentation] for more information.
```

Comment:
At some point we might want to update this link to point to https://www.elastic.co/guide/en/apm/agent/js-base/master/distributed-tracing-guide.html - but that page hasn't yet been published outside of master, so I don't think we can link to it yet without the doc build complaining